### ISSUE TITLE
Capacity Utilization reports wrong usage

### ISSUE THREAD

**Issue by djgalloway** [ceph/cephmetrics#104]
I was looking at `ceph df` output on the Sepia cluster because we've maxed it out a few times over the past month and noticed cephmetrics is reporting the RAW USED percentage.

I think the better method would be to sum the USED% of all the RBD pools.

When the cluster was completely full and preventing writes last week, all 4 pools reported 100% usage.

```
root@mira021:~# ceph df
GLOBAL:
    SIZE     AVAIL      RAW USED     %RAW USED 
    274T     84692G         191T         69.87 
POOLS:
    NAME             ID     USED       %USED     MAX AVAIL     OBJECTS  
    data             0      64768G     97.56         1599G     49996056 
    metadata         1        675M      0.05         1199G      2006049 
    libvirt-pool     4       2048M      0.12         1599G          518 
    rbd              11       210M      0.01         1599G          237 
```

**Comment by pcuzner** [comment]
I think the at-a-glance view of raw consumption is OK - but cephmetrics should to drill down further to expose pool level capacity, and add a corresponding alert. Flagging this as an RFE

**Comment by zmc** [comment]
Perhaps we should look at merging the `pools` subdict of `ceph df -f json` into the `pools` subdict of `Mon.get_stats()`

**Comment by pcuzner** [comment]
+1 that gets the data, then we need to visualise and add the alert in place. I'll add a task to the to-do board to reflect what needs to be done.

**Comment by zmc** [comment]
- [ ] update the mon collector to gather pool usage metrics
- [ ] add/modify dashboard 
- [ ] add pool usage alert definition

**Comment by pcuzner** [comment]
resolved in #112
